packages/frontend-shared/src/locales/en-US.json

```diff
@@ -357,7 +357,11 @@
       "languageLabel": "Language",
       "configFileLanguageLabel": "Cypress config file",
       "detected": "(detected)",
-      "browseIntegrations": "Browse our list of third-party framework integrations"
+      "browseIntegrations": "Browse our list of third-party framework integrations",
+      "communityFrameworkDefinitionProblem": "Community framework definition problem",
+      "communityDependenciesCouldNotBeParsed": "This project has a community framework definition installed that could not be parsed. It is located at the following path: | This project has some community framework definitions installed that could not be parsed. They are located at the following paths:",
```

Comment:
Was this language suggested by design/UX? Wonder whether it would be better to use less technical lingo - ie: "that could not be loaded" instead of "parsed"

Reply:
It wasn't suggested by them but it was validated with them. I guess loaded/parsed are both fine, happy to switch to loaded.